src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+    private static final String[] Colors = {"red", "blue", "yellow", "green", "purple", "black"};

```

Comment:
The array `Colors` should follow the Java naming convention for constants, which is all uppercase letters with words separated by underscores. Consider renaming it to `COLORS`.